bin/prod

```diff
@@ -0,0 +1,53 @@
+PROD_ECR_REPO="public.ecr.aws/a5f0m6q3/syncm8_prod"
+
+# Build prod image.
+dockerBuild(){
+    bin/util logAndRun "docker build --pull --rm --target prod -t $PROD_ECR_REPO ."
+}
+
+# Push prod image to aws ecr.
+dockerPush(){
+    echo "Are you sure you want to push an image to PRODUCTION?????"
+    read ans
+    if [ $ans == "y" ]; then
+        bin/util logAndRun "docker push $PROD_ECR_REPO"
+    fi
+}
+
+# Restart AWS ECS service.
+awsDeploy(){
+    bin/infra priv "aws ecs update-service --force-new-deployment --service service --cluster backend-cluster | cat"
+}
+
+
+# Print help message
+printHelp(){
+    echo "Usage:  bin/infra COMMAND
+
+Command Runner for syncm8 infra commands.
+
+Commands:
+  build       Build prod image.
+  push        Push prod image to aws ecr.
+                  N.b. Must be authenticated with ecr.
```

Comment:
Is there a way to programmatically check if a user is authenticated?

Reply:
"authenticated with ecr" actually means you've given the docker cli aws credentials. If docker tries to push and it rejects, it will yell at you in the terminal and tell you to authenticate. 
This is a code path used primarily for one off debugging, so will get very little usage - not worth time investment to make it more user friendly imo.

Reply:
sounds good, esp if aws cli will tell you to auth